alphagov/paas-compose-broker
# [#143876117] Creation of mongodb

## What

This PR provides mongo database provisioning, binding and de-provisioning ability for cloudfoundry marketplace users. It also comes with set of tests using fake compose client and real deal integration tests that create instances. The latter are not currently executed via Travis. 

## How to review
- sanity check 
- deploy via paas-cf. Test with https://github.com/alphagov/paas-cf/pull/945
- ensure all tests are passing
- test manually by deploying an app to your dev env and binding an instance

## Who
Not...@combor, @henrytk , @paroxp , @LeePorte or @timmow
